f5router/f5router_test.go

```diff
@@ -229,7 +445,48 @@ var _ = Describe("F5Router", func() {
 			wildBeginEndpoint = makeEndpoint("127.0.6.4")
 			wild2Endpoint = makeEndpoint("127.0.7.1")
 			quxEndpoint = makeEndpoint("127.0.7.1")
+			brokerEndpoint1 = makeEndpoint("127.0.8.1")
+			brokerEndpoint2 = makeEndpoint("127.0.8.2")
+
+			// Add broker plans to router
+			plans := make(map[string]planResources.Plan)
+
+			// Updates to virtual and pool but no health monitors
+			plans["plan1"] = planResources.Plan{
+				ID: "plan1",
+				Pool: planResources.PoolType{
+					Balance: "ratio-member",
+				},
+				VirtualServer: planResources.VirtualType{
+					Policies:    []string{"/test/plan1-policy"},
+					Profiles:    []string{"/test/plan1-profile"},
+					SslProfiles: []string{"/test/plan1-ssl-profile"},
+				},
+			}
+
+			// Updates to pool health monitors only
+			plans["plan2"] = planResources.Plan{
+				ID: "plan2",
+				Pool: planResources.PoolType{
+					HealthMonitors: []bigipResources.Monitor{
+						bigipResources.Monitor{
+							Name:     "plan2-monitor1",
+							Protocol: "http",
```

Comment:
need to set partition in monitor type

Reply:
When creating custom health monitors need to use joinBigipPath with the name and config partition otherwise if the monitors already exist don't touch the name.

Reply:
Attach to pool = use joinBigipPath; create = don't touch the name.

Reply:
Fixed.